index.html

```diff
@@ -3,51 +3,50 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>myproject</title>
+    <title>Document</title>
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
-    <header>
-        <nav >
-            <div class="menu-container">
-                <ul>
-                    <li><a href="#">My logo</a></li>
-                    <li style="place-items: left;"><a href="#",></a>
-                        <div class="bar"></div>
-                        <div class="bar"></div>
-                        <div class="bar"></div>
-                    </li>
-                </ul>
+    <div class="body">
+        <div class="content">
+            <img src="c:\Users\kalyst\Downloads\Microverse-Student-Project-1 (1)\Bars\Status Bars\Dark.png">
```

Comment:
To deal with images in your project I advise to Crete a separate folder, for the images inside your project so that your project to be organized.